### DISCUSSION TITLE
Can't filter discussions on Android

### DISCUSSION THREAD

**Discussion by RobinLe0** [community/community#141319] [Mobile]
I can enter text in the search field, but pressing the search button does nothing

**Comment by alcere** [comment]
Hi @RobinLe1402. Thanks for reporting this problem to us. I was able to reproduce the issue on my end, and I've notified the Android team. We are sorry for the troubles!

**Reply by Igorko777** [reply]
Android 14

**Reply by rona901** [reply]
> Android 14
> 
🔙🆙
